sections/04_study.md

```diff
@@ -389,7 +390,7 @@ align sequences and predict bulges, mismatches, and wobble base pairing without
 requiring the user to input secondary structure predictions or thermodynamic
 calculations.
 
-Further incremental advances in neural-network approaches for miRNA and target
+Further incremental advances in neural network approaches for miRNA and target
```

Comment:
Are suggesting we replace NN with DL here?  I've left NN when the original authors used it.  I'm not sure if they are using a DL synonym or referring to the pre-DL days.